lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -666,8 +664,52 @@ def infer(self, images: List[Image.Image], threshold: float) -> List[List[Elemen
                 )
                 elements.append(element)
             batched_results.append(elements)
+            if self.cache:
+                key = Cache.get_hash_key(image.tobytes())
```

Comment:
Should we hash-in some representation of the software version or processing options?

Reply:
the model just uses threshold! So hashing-in threshold for now.